### PR TITLE
Hello React Fron end

### PR DESCRIPTION
**Pull request by luccamicale**
## Hello Rails Back End  SetUp

##https://github.com/luccamicale/hello-react-frontend

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I created a react app
- [ ] follow the requirements

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Greeting from './pages/Greetings';
+
+function App() {
+  return (
+    <Greeting />
+  );
+}
```
Comment (mikethreels) [inline src/App.js]: - [ ] the App component should have a route that leads to the Greeting component.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Lucca, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
I modified the only requierement for the last reviewer. ow my project should be approved!!
💯 👍 🔢 🥇 

routess

**Review by mckent05 — APPROVED** [review]
Hi @luccamicale, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job, both apps work as expected and communicate with each other 👏 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**